tests/integration/security/testdata/jwt/simple-jwt-policy.yaml.tmpl

```diff
@@ -1,5 +1,5 @@
-apiVersion: "authentication.istio.io/v1alpha1"
-kind: "Policy"
+apiVersion: "security.istio.io/v1beta1"
+kind: "RequestAuthentication"
```

Comment:
Could you create new files for RequestAuthentication so that we don't lose any existing tests?